tests/everflow/everflow_test_utilities.py

```diff
@@ -44,17 +44,13 @@ def setup_info(duthost, tbinfo):
         dict: Required test information
 
     """
-    # TODO: Support all T1 and T0 topos in these tests.
-    if tbinfo["topo"]["name"] not in ("t1", "t1-lag", "t1-64-lag", "t1-64-lag-clet"):
-        pytest.skip("Unsupported topology")
```

Comment:
Why is this removed?

Reply:
It's because all the everflow test scripts already have the topology mark:
```python
pytestmark = [
    pytest.mark.topology("t1")
]
```
It's unnecessary to explicitly check current topology type in setup section.

Reply:
missed it.